### DISCUSSION TITLE
Switching Repositories filter should show spinner

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44028] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

GitHub Mobile Beta

1. Open the repositories view (as [me](https://api.github.com/users/jsoref), I have a couple of repositories... and one of my orgs has [a couple](https://api.github.com/orgs/check-spelling))
2. Select the `Fork` filter
3. Tap the filter again and select `Source` instead
4. Wait as the display just sits there for ~5 seconds showing the stale results

### Expected

Show a spinner if it will take >1s to load replacement matches.

**Comment by HenSquared** [comment, marked as answer]
Good find. I've opened an issue to get this fixed on Android!
